gui/templates/system/system_info.html

```diff
@@ -102,6 +102,10 @@ <h1>{% trans "System Information" %}</h1>
         <th>{% trans "System Product" %}</th>
         <td class="descr">{{ local.system_product }}</td>
     </tr>
+    <tr>
+        <th>{% trans "License" %}</th>
+        <td class="descr">{{ local.license }}</td>
+    </tr>
```

Comment:
What if there is no license? Should we encapsulate that block within {% if license %}?

Reply:
Or should we display red text "No license"?

Reply:
License is very rare on FreeNAS. Perhaps show "No license" for TrueNAS and hide it from FreeNAS?

Reply:
OK. It's already hidden from FreeNAS btw (see full context)